zavod/zavod/shed/fsf.py

```diff
@@ -58,12 +58,11 @@ def parse_sanctions(context: Context, entity: Entity, entry: Element) -> None:
     for regulation in regulations:
         url = regulation.findtext("./publicationUrl")
         assert url is not None, etree.tostring(regulation)
-        program = regulation.get("programme")
+        source_program_key = regulation.get("programme")
         sanction = h.make_sanction(
             context,
             entity,
-            program=program,
-            program_key=program,
```

Comment:
is replacing the source-provided program string we've been publishing in `Sanction:program` for some time with our `programId` and `program.title` something we need to take through a scheduled and pre-announced release process? In case customers have logic on this on their end

Reply:
Do we want to keep the source-provided keys somewhere, e.g. Sanction:unscId. Should we introduce a sanctionTitle or sanctionLabel property instead?

Reply:
I simplified this, we're now no longer putting in our title.

Do you think it's worth keeping around the original value that we generated our `program_key` from?

Reply:
Well, I think you had a strong-ish opinion about this and I don't, so I'll put it in there.